src/intents/index.js

```diff
@@ -61,7 +61,7 @@ export async function getRedirectionURL(cozy, type, data) {
   // await deleteIntent(cozy, intent)
 
   // ignore query string and intent id
-  const baseURL = service.href.split('?')[0]
+  const baseURL = service.href.replace(/\?[\w-=&]*/, '')
```

Comment:
I do not understand what this is supposed to do :/ I think naming the regex should help.

Can you also add test cases ?

Reply:
By requesting an intent from the stack, the stack will provide an intent ID like `http://store.cozy.tools:8080/?intent=85ef0631669a4ea34ecf9ca6d10298d5#/redirect`. Previously, the hash part was before the query and the previous code worked. But now the hash part is provided at the end, it's why we use this regex to remove the query part only.

Good point. I updated the tests, thanks 👍